spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -10,10 +18,32 @@
     end
 
     context "when sending a welcome_thread notification" do
-      xit "generates the appropriate broadcast to be sent to a user"
-      xit "it sends a welcome notification for that broadcast"
-      xit "it does not send duplicate welcome notification for that broadcast"
-      xit "does not send a notification to a user who has commented in a welcome thread"
+      let(:welcome_thread_article) { create(:article, title: "Welcome Thread") }
+      let(:welcome_thread_comment) { create(:comment, commentable: welcome_thread_article, user: user) }
```

Comment:
I'm confused about why we're creating `welcome_thread_article` and `welcome_thread_comment` here. They're not being used at all 🤔 Although, I would expect that we would be testing to see that someone who has commented on a welcome thread doesn't get a notification...

Reply:
Good catch! I've since fixed this. 😄